react-accordion-component/accordion/src/App.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useState } from 'react';
+import './App.css';
+
+const topics = [
+  {
+    id: 1,
+    title: 'Hypertext Markup Language',
+    content:
+      'Hypertext Markup Language (HTML) is the standard ... for the World Wide Web',
+  },
+  {
+    id: 2,
+    title: 'Cascading Style Sheets',
+    content:
+      'Cascading Style Sheets (CSS) is a style sheet ... of the World Wide Web',
+  },
+  {
+    id: 3,
+    title: 'Javascript',
+    content:
+      'Javascript, often abbreviated as JS, is ... and first-class functions',
+  },
+];
+
+type Topic = {
+  id: number;
+  title: string;
+  content: string;
+};
+
+type AccordionProps = {
+  topics: Topic[];
+};
+
+function App() {
+  return <Accordion topics={topics} />;
+}
+
+const Accordion: React.FC<AccordionProps> = ({ topics }) => {
```

Comment:
The React.FC type is now discouraged (though there are arguments resurfacing about it). You can trivially convert this, though:
`const Accordion = ({ topics }: AccordionProps) => {`

I still prefer the function form, though, but it's a personal preference:
`function Accordion({ topics }: AccordionProps) {`